apps/web/composables/useQuickCheckout/useQuickCheckout.ts

```diff
@@ -58,8 +59,9 @@ export const useQuickCheckout: UseQuickCheckoutReturn = () => {
    * openQuickCheckout();
    * ```
    */
-  const openQuickCheckout: OpenQuickCheckout = (product: Product) => {
```

Comment:
I dont get how this logic works, wouldnt this always override the quantity value. 
If I add a product with quantity 2 and then add the same product again with 2. The quantity value would show 2 and not 4 right?

Reply:
Also, I would like to see a unit test for this behavior.

Reply:
for what i understood overlay quantity should display last added quantity and product, but you ar right that it makes sens to take into consideration previous quantities